frontend/src/api/TokenRefreshInstance.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
File name bit confusing: it does not contain `TokenRefresh` instance, only implementation.
General note on this file: I think we can consolidate it with `authRequest` with renaming into `AuthApiClient`. Functional of `TokenRefresh` may be the part of `AuthApiClient`